src/ceph-volume/ceph_volume/devices/lvm/activate.py

```diff
@@ -111,6 +112,8 @@ def get_osd_device_path(osd_lv, lvs, device_type, dmcrypt_secret=None):
 def activate_bluestore(lvs):
     # find the osd
     osd_lv = lvs.get(lv_tags={'ceph.type': 'block'})
+    if not osd_lv:
```

Comment:
This is similar to what happens in `activate_filestore` but we didn't have it here and I noticed it with this failure:

    File "/usr/lib/python2.7/dist-packages/ceph_volume/devices/lvm/activate.py", line 115, in activate_bluestore
      is_encrypted = osd_lv.tags.get('ceph.encrypted', '0') == '1'
    AttributeError: 'NoneType' object has no attribute 'tags'

This happened because we were missing the `--filestore` flag, so it defaulted to bluestore, where no OSDs could be found. This change should be backporter